Components/scripts/TestMap.gd

```diff
@@ -73,10 +73,10 @@ func add_room(name, room, wall):
 	
```

Comment:
Is this file even needed anymore?

Reply:
It could be archived, I think it is still useful just as a resource to see how the map system is used. Perhaps we could rename it to ExampleMap.txt ?

Reply:
Fair enough, yeah I think renaming them may be a good idea